### ISSUE TITLE
Ignored return value of token.mint in Crowdsale contract

### ISSUE THREAD

**Issue by spalladino** [OpenZeppelin/zeppelin-solidity#730]
- [x] 🐛 This is a bug report.
- [ ] 📈 This is a feature request.

Function `mint` from MintableToken returns a boolean value, similar to `transfer`. However, its return value is not checked when the token is [used in the Crowdsale contract](https://github.com/OpenZeppelin/zeppelin-solidity/blob/master/contracts/crowdsale/Crowdsale.sol#L78).

Instead of:
```
token.mint(beneficiary, tokens);
```
Should be:
```
require(token.mint(beneficiary, tokens));
```

On the other hand, the implementation of mint does not return false, and instead should revert if it fails. 

Discuss whether we want to remove the return value from `mint`, or we want to check it whenever we invoke the method.

**Comment by fiiiu** [comment]
This has now moved to [the MintedCrowdsale](zeppelin-solidity/contracts/crowdsale/emission/MintedCrowdsale.sol) under the crowdsale refactor #744.

I'm not fixing this quite yet in hope for more input regarding the `mint` return value.. Any reason for keeping it? (See also `finishMinting` in `MintableToken`).

**Comment by spalladino** [comment]
The main reason for keeping it would be consistency with the `transfer` method (you can consider mint as a transfer from address zero, which is reflected in the Transfer event emitted). My vote would be to keep the boolean, and check the return value in the crowdsale.

**Comment by fiiiu** [comment]
Understood, pushed fix for this in the crowdsale refactor #744.
